### PR TITLE
Added build-scripts for 7 packages.

### PR DESCRIPTION
**Pull request by varsha-aaynure**
Created build-scripts for these 7 packages:
1. babel-plugin-transform-runtime
2. jackson-dataformat-csv
3. jackson-module-scala
4. netty-transport-native-epoll
5. picocli
6. vertx-web
7. vertx-web-graphql

### CODE REVIEW

File: n/netty-transport-native-epoll/netty-transport-native-epoll_ubi8_3.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/netty/netty.git
+PACKAGE_VERSION=netty-4.1.60.Final
```
Comment (kandarpamalipeddi) [inline n/netty-transport-native-epoll/netty-transport-native-epoll_ubi8_3.sh:22]: Can you replace this with `PACKAGE_VERSION=${1:-netty-4.1.60.Final}` ?

File: n/netty-transport/netty-transport_v4.1.48_ubi8_3.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/netty/netty.git
+PACKAGE_VERSION=netty-4.1.48.Final
```
Comment (kandarpamalipeddi) [inline n/netty-transport/netty-transport_v4.1.48_ubi8_3.sh:22]: Can you replace this with `PACKAGE_VERSION=${1:-netty-4.1.48.Final}` ?

File: n/netty-transport/netty-transport_v4.1.60_ubi8_3.sh [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/netty/netty.git
+PACKAGE_VERSION=netty-4.1.60.Final
+
+echo "Usage: $0 [<PACKAGE_VERSION>]"
+echo "PACKAGE_VERSION is an optional parameter whose default value is netty-4.1.60.Final, not all versions are supported."
+
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
```
Comment (kandarpamalipeddi) [inline n/netty-transport/netty-transport_v4.1.60_ubi8_3.sh:27]: Can you do this at line 22 ?

File: n/node-proper-lockfile/node-proper-lockfile_ubi8_3.sh [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+#Variables
+PACKAGE_URL=https://github.com/moxystudio/node-proper-lockfile.git
+PACKAGE_VERSION=v4.1.1
+NODE_VERSION=v12.22.4
+
+echo "Usage: $0 [<PACKAGE_VERSION>]"
+echo "PACKAGE_VERSION is an optional parameter whose default value is v4.1.1, not all versions are supported."
+
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
```
Comment (kandarpamalipeddi) [inline n/node-proper-lockfile/node-proper-lockfile_ubi8_3.sh:28]: Can you do this at line 22 ?

File: p/picocli/picocli_v4.5.2_ubi8_3.sh [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/remkop/picocli.git
+PACKAGE_VERSION=v4.5.2
```
Comment (kandarpamalipeddi) [inline p/picocli/picocli_v4.5.2_ubi8_3.sh:22]: Can you assign as : `PACKAGE_VERSION=${1:-v4.5.2}` ?

File: v/vertx-web-api-contract/vertx-web-api-contract_ubi8_3.sh [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/vert-x3/vertx-web.git
+PACKAGE_VERSION=4.1.2
+
+echo "Usage: $0 [<PACKAGE_VERSION>]"
+echo "PACKAGE_VERSION is an optional parameter whose default value is 4.1.2, not all versions are supported."
+
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
```
Comment (kandarpamalipeddi) [inline v/vertx-web-api-contract/vertx-web-api-contract_ubi8_3.sh:27]: Can you do this at line 22 ?

File: v/vertx-web-client/vertx-web-client_v3.9.7_ubi8_3.sh [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/vert-x3/vertx-web.git
+PACKAGE_VERSION=3.9.7
+
+echo "Usage: $0 [<PACKAGE_VERSION>]"
+echo "PACKAGE_VERSION is an optional parameter whose default value is 3.9.7, not all versions are supported."
+
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
```
Comment (kandarpamalipeddi) [inline v/vertx-web-client/vertx-web-client_v3.9.7_ubi8_3.sh:27]: Can you do this at line 22 ?

File: v/vertx-web-client/vertx-web-client_v4.1.2_ubi8_3.sh [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+
+#Variables
+PACKAGE_URL=https://github.com/vert-x3/vertx-web.git
+PACKAGE_VERSION=4.1.2
+
+echo "Usage: $0 [<PACKAGE_VERSION>]"
+echo "PACKAGE_VERSION is an optional parameter whose default value is 4.1.2, not all versions are supported."
+
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
```
Comment (kandarpamalipeddi) [inline v/vertx-web-client/vertx-web-client_v4.1.2_ubi8_3.sh:28]: Can you do this at line 22 ?

File: v/vertx-web-graphql/vertx-web-graphql_ubi8_3.sh [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/vert-x3/vertx-web.git
+PACKAGE_VERSION=4.1.2
```
Comment (kandarpamalipeddi) [inline v/vertx-web-graphql/vertx-web-graphql_ubi8_3.sh:22]: Can you assign it like : `PACKAGE_VERSION=${1:-4.1.2}`?

File: v/vertx-web/vertx-web_v3.9.1_ubi8_3.sh [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/vert-x3/vertx-web.git
+PACKAGE_VERSION=3.9.1
```
Comment (kandarpamalipeddi) [inline v/vertx-web/vertx-web_v3.9.1_ubi8_3.sh:22]: Can you assign it like : PACKAGE_VERSION=${1:-3.9.1}?

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]
Minor comments.

File: a/apicurio-registry-common/apicurio-registry-common_ubi8_3.sh [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/Apicurio/apicurio-registry.git
+PACKAGE_VERSION=1.3.0.Final
+
+echo "Usage: $0 [<PACKAGE_VERSION>]"
+echo "PACKAGE_VERSION is an optional parameter whose default value is 1.3.0.Final, not all versions are supported."
+
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
```
[the remark this answers is not in the document]
Comment (varsha-aaynure) [reply a/apicurio-registry-common/apicurio-registry-common_ubi8_3.sh:27]: yes.

File: b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+
+yum update -y 
+
+#Install required files
+yum install -y git 
+
+#To install node 12 
+wget "https://nodejs.org/dist/v12.22.4/node-v12.22.4-linux-ppc64le.tar.gz"
+tar -xzf node-v12.22.4-linux-ppc64le.tar.gz
+export PATH=$CWD/node-v12.22.4-linux-ppc64le/bin:$PATH
```
[the remark this answers is not in the document]
Comment (varsha-aaynure) [reply b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh:32]: It is a variable $cwd .

[the remark this answers is not in the document]
Comment (varsha-aaynure) [reply b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh:32]: Checked this is working without this variable, so now I'm going to remove it from the script.

File: b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/babel/babel.git
+PACKAGE_VERSION=v7.4.0
```
[the remark this answers is not in the document]
Comment (varsha-aaynure) [reply b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh:22]: Yes.

File: e/express-static-gzip/express-static-gzip_ubi8_3.sh [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+
+#Variables
+NODE_VERSION=v12.22.4
+PACKAGE_VERSION=v0.3.2
+PACKAGE_URL=https://github.com/tkoenig89/express-static-gzip.git
+
+echo "Usage: $0 [<PACKAGE_VERSION>]"
+echo "PACKAGE_VERSION is an optional parameter whose default value is v0.3.2, not all versions are supported."
+
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
```
[the remark this answers is not in the document]
Comment (varsha-aaynure) [reply e/express-static-gzip/express-static-gzip_ubi8_3.sh:28]: Yes.

File: j/jackson-dataformat-csv/jackson-dataformat-csv_ubi8_3.sh [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/FasterXML/jackson-dataformats-text.git
+PACKAGE_VERSION=jackson-dataformats-text-2.10.5
```
[the remark this answers is not in the document]
Comment (varsha-aaynure) [reply j/jackson-dataformat-csv/jackson-dataformat-csv_ubi8_3.sh:22]: PACKAGE_VERSION=${1:-jackson-dataformats-text-2.10.5} added in script.

File: v/vertx-web-api-contract/vertx-web-api-contract_ubi8_3.sh (hunk 6 above)
Comment (varsha-aaynure) [reply v/vertx-web-api-contract/vertx-web-api-contract_ubi8_3.sh:27]: Yes.

File: v/vertx-web-client/vertx-web-client_v3.9.7_ubi8_3.sh (hunk 7 above)
Comment (varsha-aaynure) [reply v/vertx-web-client/vertx-web-client_v3.9.7_ubi8_3.sh:27]: Yes.

File: v/vertx-web-client/vertx-web-client_v4.1.2_ubi8_3.sh (hunk 8 above)
Comment (varsha-aaynure) [reply v/vertx-web-client/vertx-web-client_v4.1.2_ubi8_3.sh:28]: Yes.

File: v/vertx-web-graphql/vertx-web-graphql_ubi8_3.sh (hunk 9 above)
Comment (varsha-aaynure) [reply v/vertx-web-graphql/vertx-web-graphql_ubi8_3.sh:22]: Yes.

File: v/vertx-web/vertx-web_v3.9.1_ubi8_3.sh (hunk 10 above)
Comment (varsha-aaynure) [reply v/vertx-web/vertx-web_v3.9.1_ubi8_3.sh:22]: Yes.

**Comment by kandarpamalipeddi** [comment]
@varsha-aaynure 
Can you please make required changes, so that I can merge this PR.
CC: @seth-priya @amitsadaphule

**Comment by varsha-aaynure** [comment]
yes sure @kandarpamalipeddi

**Pull request closed without merging**